src/client/src/orangehrmPerformancePlugin/pages/EmployeeTrackerLogs.vue

```diff
@@ -0,0 +1,529 @@
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <div class="orangehrm-employee-tracker">
+        <div class="orangehrm-employee-tracker-image-section">
+          <div class="orangehrm-employee-tracker-image-wrapper">
+            <div class="orangehrm-employee-tracker-image">
+              <img
+                alt="profile picture"
+                class="employee-image"
+                :src="employeeImgSrc"
+              />
+            </div>
+          </div>
+        </div>
+        <div class="orangehrm-employee-tracker-header-section">
+          <div class="orangehrm-employee-tracker-header">
+            <oxd-text tag="h5" class="orangehrm-employee-tracker-header-title">
+              {{ trackerName }}
+            </oxd-text>
+            <oxd-text
+              tag="h6"
+              class="orangehrm-employee-tracker-header-subtitle"
+            >
+              {{ employeeName }}
+            </oxd-text>
+          </div>
+          <div class="orangehrm-employee-tracker-ratings">
+            <div
+              v-if="meta.positive > 0"
+              class="orangehrm-employee-tracker-ratings-info"
+            >
+              <div class="orangehrm-employee-tracker-ratings-icon --positive">
+                <oxd-icon name="hand-thumbs-up-fill" />
+              </div>
+              <oxd-text
+                class="orangehrm-employee-tracker-ratings-text --positive"
+              >
+                {{ meta.positive }}
+              </oxd-text>
+            </div>
+            <div
+              v-if="meta.negative > 0"
+              class="orangehrm-employee-tracker-ratings-info"
+            >
+              <div class="orangehrm-employee-tracker-ratings-icon --negative">
+                <oxd-icon name="hand-thumbs-down-fill" />
+              </div>
+              <oxd-text
+                class="orangehrm-employee-tracker-ratings-text --negative"
+              >
+                {{ meta.negative }}
+              </oxd-text>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <br />
+    <div class="orangehrm-paper-container">
+      <div class="orangehrm-header-container">
+        <oxd-text tag="h5" class="orangehrm-employee-tracker-log-list-header">
+          {{ $t('performance.tracker_logs') }}
+        </oxd-text>
+        <oxd-button
+          :label="$t('performance.add_log')"
+          icon-name="plus"
+          display-type="secondary"
+          @click="onClickAdd"
+        />
+      </div>
+      <oxd-grid ref="scrollerRef" :cols="1" class="orangehrm-container">
+        <oxd-sheet
```

Comment:
What if we move this card to a separate component? might save some clarity here

Reply:
moved to a separate component